monkey/monkey_island/cc/services/authentication_facade/flask_resources/register_resources.py

```diff
@@ -0,0 +1,20 @@
+import flask_restful
+
+from common import DIContainer
+
+from ..authentication_service import AuthenticationService
+from .login import Login
+from .logout import Logout
+from .register import Register
+from .registration_status import RegistrationStatus
+
+
+def register_resources(api: flask_restful.Api, container: DIContainer):
+    authentication_service = container.resolve(AuthenticationService)
+
+    api.add_resource(Register, *Register.urls, resource_class_args=(authentication_service,))
+    api.add_resource(
+        RegistrationStatus, *RegistrationStatus.urls, resource_class_args=(authentication_service,)
+    )
+    api.add_resource(Login, *Login.urls, resource_class_args=(authentication_service,))
+    api.add_resource(Logout, *Logout.urls, resource_class_args=(authentication_service,))
```

Comment:
We need to investigate if this code is necessary or if it can be replaced with a blueprint. I've created an issue here: https://github.com/guardicore/monkey/issues/3086

We should address it before doing more consolidations like this

Reply:
+1